hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/ha/SCMHAManagerImpl.java

```diff
@@ -56,7 +69,28 @@ public void start() throws IOException {
    */
   @Override
   public boolean isLeader() {
-    return isLeader;
+    if (!SCMHAUtils.isSCMHAEnabled(conf)) {
+      // When SCM HA is not enabled, the current SCM is always the leader.
+      return true;
+    }
```

Comment:
Add a todo please
```
// will change from RaftServerImpl:::isLeader() to RaftServerImpl::getRoleInfoProto()
// after RATIS-1001 is merged.
```

Reply:
https://issues.apache.org/jira/browse/HDDS-3962 
Created a JIRA to track this.